autoscale/src/main/java/com/sequenceiq/periscope/service/RegularScalingAdjustmentService.java

```diff
@@ -95,13 +109,22 @@ private List<String> collectRegularDownscaleRecommendations(Cluster cluster, Map
             int allowedDownscale = IntStream.of(mandatoryDownscaleCount, maxAllowedDownscale,
                     loadAlertConfiguration.getMaxScaleDownStepSize()).min().getAsInt();
 
+            long now = Instant.now().toEpochMilli();
             YarnScalingServiceV1Response yarnResponse = yarnMetricsClient.getYarnMetricsForCluster(cluster, stackResponse, policyHostGroup,
                     pollingUserCrn, Optional.of(mandatoryDownscaleCount));
             List<String> yarnRecommendedDecommissionHosts =
                     yarnResponseUtils.getYarnRecommendedDecommissionHostsForHostGroup(yarnResponse, hostFqdnsToInstanceId);
-            return yarnRecommendedDecommissionHosts.stream()
+            List<String> finalDecommissionHosts = yarnRecommendedDecommissionHosts.stream()
                     .limit(allowedDownscale)
                     .collect(Collectors.toList());
+
+            String metricsCollectionMessage = messagesService.getMessageWithArgs(AUTOSCALING_METRICS_COLLECTION_DECOMMISSION_RECOMMENDATION,
```

Comment:
This is not publishing to the activity console, correct? (More specifically: no part of this patch is changing any messages to the activity console?). Instead, it's using the MessageService as a formatting mechanism (why not use a regular Sting.format - especially if this information is not user visible)?